### PR TITLE
feat: monitor bulk ann downloads

### PR DESCRIPTION
**Pull request by pedrokohler**
Addresses: https://github.com/ImagingDataCommons/slim/issues/235

### CODE REVIEW

File: src/viewer.js [hunk 1]
```diff
@@ -3362,6 +3362,8 @@ class VolumeImageViewer {
        * ellipse or rectangle is counted as one Annotation.
        */
       const numberOfAnnotations = Number(metadataItem.NumberOfAnnotations)
+      console.log('AnnotationGroupUID:', metadataItem.AnnotationGroupUID, 'NumberOfAnnotations:', numberOfAnnotations)
```
Comment (igoroctaviano) [inline src/viewer.js:3365]: Do we need this log?

**Review by igoroctaviano — CHANGES_REQUESTED** [review]

File: src/viewer.js (hunk 1 above)
Comment (pedrokohler) [reply src/viewer.js:3365]: Yes

Comment (igoroctaviano) [inline src/viewer.js:3365]: Please use debug

File: src/annotation.js [hunk 2]
```diff
@@ -235,17 +235,25 @@ async function _fetchGraphicData ({
         `Could not find bulkdata of annotation group "${uid}".`
       )
     } else {
+      const progressCallback = (progressEvent) => {
+        console.info(`Loaded ${Math.round(progressEvent.loaded / 1024 / 1024 * 100) / 100} MB from annotation group "${uid}"`)
```
Comment (igoroctaviano) [inline src/annotation.js:239]: Please use debug

**Review by igoroctaviano — CHANGES_REQUESTED** [review]

File: src/viewer.js (hunk 1 above)
Comment (pedrokohler) [reply src/viewer.js]: Done.

**Review by igoroctaviano — APPROVED** [review]
LGTM

**Pull request merged**